### ISSUE TITLE
Conjunctival foreign body sensation

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9863]
**Preferred term label:**
Conjunctival foreign body sensation

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Foreign body in the external eye

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@Sukram @MickeySegal 
Is this distinct from Corneal foreign body sensation or should we rename the latter (new) term to Conjunctival foreign body sensation

**Comment by MickeySegal** [comment]
I'm checking with the doctor who created both these terms.  It seems they should be clarified or merged.

**Comment by pnrobinson** [comment]
@MickeySegal 
any update?

**Comment by MickeySegal** [comment]
We merged the terms as "eye foreign body sensation or observation" because of the difficulty of distinguishing between corneal and conjunctival sensation.  We are using https://hpo.jax.org/app/browse/term/HP:0034804, which perhaps can get some additional language.

**Comment by pnrobinson** [comment]
@MickeySegal 
added Conjunctival foreign body sensation and Foreign body sensation in the eye as synonyms.
